rhobs/alerting/data_plane/prometheus.pipeline_to_snapshot_alerts.yaml

```diff
@@ -28,7 +28,7 @@ spec:
           PipelineRunFinish to SnapshotInProgress time exceeded
         description: >
           Time from pipeline run finished to snapshot marked in progress has been over
-          30s for more than 10% of requests during the last 5 minutes on cluster
+          30s for more than 10% of requests during the last 10 minutes on cluster
```

Comment:
We're looking now at a time window of 10 minutes backward and we require it to fulfill a condition continuously for 10 minutes.
This is different comparing to what we had before in which we looked at a window of 5 minutes and we required the condition to be consistent for 1 minute.

The change in the description might need to reflect that.

Reply:
Ack, updated the description to account for the condition to be failing continuously for 10 minutes.